workspaces/ui/src/components/diff/v2/CaptureManagerPage.js

```diff
@@ -656,16 +655,15 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     flexGrow: 1,
     flexShrink: 1,
-    overflow: 'scroll',
-    height: '100vh',
     justifyContent: 'center',
   },
   navRoot: {
     flexGrow: 1,
     position: 'fixed',
     width: 'inherit',
     height: '100vh',
-    overflowY: 'scroll',
+    overflowY: 'visible',
```

Comment:
I think this is where the left over problem you've described comes from: while we want the normal page content to be in the main scroll context, the left hand nav can have it's own! Therefor, it should be fine for this one to have a `overflowY: scroll`. When set to `visible` it will just bleed past the container. Probably not a big deal with the small amount of content (overflow is pretty unlikely), but perhaps a fix worth trying regardless.

Reply:
doesn't fix anything :| no matter which one, it still results in being unable to scroll to the two tabs. I'm actually going to keep with visible, because otherwise it adds a scrollbar at all times there, and it looks kind of ugly

https://www.loom.com/share/32dab57277b04a6580d91fa68126986a